test/powershell/Modules/Microsoft.PowerShell.Utility/Get-Date.Tests.ps1

```diff
@@ -24,6 +24,11 @@ Describe "Get-Date DRT Unit Tests" -Tags "CI" {
         Get-date -Date:"Jan 1, 2020"  -Format:"MMM-dd-yy" | Should -Be "Jan-01-20"
     }
 
+    It "using -AsUTC produces the correct output" {
+        Get-date -Date:"2020-01-01T00:00:00" -Format:"MMM-dd-yy hh:mm" | Should -Be "Jan-01-20 12:00"
+        Get-date -Date:"2020-01-01T00:00:00" -Format:"MMM-dd-yy hh:mm" -AsUTC | Should -Be "Jan-01-20 08:00"
```

Comment:
Perhaps follow will be more reliable test:
```powershell
(Get-date -Date:"2020-01-01T00:00:00").Kind | Should -Be [DateTimeKind]::Local
(Get-date -Date:"2020-01-01T00:00:00" -AsUTC).Kind | Should -Be [DateTimeKind]::Utc
```

Reply:
Done
(I had to modify the test a little from what you wrote, but I think it captures the same thing)